content/chat/rooms/index.textile

```diff
@@ -84,29 +92,42 @@ blang[javascript].
   ```[javascript]
   const room = chatClient.rooms.get('basketball-stream', {typing: {timeoutMs: 5000}});
```

Comment:
Doesn't it duplicates L80 sample? @m-hulbert

Reply:
One shows using the default values that are configured, and the other setting custom values.

Reply:
Default is line 91, custom are lines 76 and 97, both with `{timeoutMs: 5000}`. Am I missing something?